src/transformers/models/dpt/configuration_dpt.py

```diff
@@ -137,11 +149,51 @@ def __init__(
         auxiliary_loss_weight=0.4,
         semantic_loss_ignore_index=255,
         semantic_classifier_dropout=0.1,
+        backbone_featmap_shape=[1, 1024, 24, 24],
+        neck_ignore_stages=[0, 1],
+        backbone_config=None,
         **kwargs
     ):
         super().__init__(**kwargs)
 
         self.hidden_size = hidden_size
+
+        if embedding_type not in ["patch_embedding", "hybrid"]:
+            raise ValueError("Embedding type must be one of ['patch_embedding', 'hybrid']")
```

Comment:
Are we expecting more types in the future? If there is only two types, a boolean flag would be an easier API.

Reply:
Indeed we're not expecting other types